src/tape/ensemble_frame.py

```diff
@@ -0,0 +1,219 @@
+import dask.dataframe as dd
+
+from packaging.version import Version
+import dask
+DASK_2021_06_0 = Version(dask.__version__) >= Version("2021.06.0")
```

Comment:
We could also consider just requiring Dask >=2022.06.0

Reply:
Done.